src/tpl/email.mst

```diff
@@ -0,0 +1,342 @@
+      @media only screen and (max-width: 620px) {
+        table[class=body] h1 {
+          font-size: 28px !important;
+          margin-bottom: 10px !important; }
+        table[class=body] p,
+        table[class=body] ul,
+        table[class=body] ol,
+        table[class=body] td,
+        table[class=body] span,
+        table[class=body] a {
+          font-size: 16px !important; }
+        table[class=body] .wrapper,
+        table[class=body] .article {
+          padding: 10px !important; }
+        table[class=body] .content {
+          padding: 0 !important; }
+        table[class=body] .container {
+          padding: 0 !important;
+          width: 100% !important; }
+        table[class=body] .main {
+          border-left-width: 0 !important;
+          border-radius: 0 !important;
+          border-right-width: 0 !important; }
+        table[class=body] .btn table {
+          width: 100% !important; }
+        table[class=body] .btn a {
+          width: 100% !important; }
+        table[class=body] .img-responsive {
+          height: auto !important;
+          max-width: 100% !important;
+          width: auto !important; }}
+
+      /* -------------------------------------
+          PRESERVE THESE STYLES IN THE HEAD
+      ------------------------------------- */
+      @media all {
+        .ExternalClass {
+          width: 100%; }
+        .ExternalClass,
+        .ExternalClass p,
+        .ExternalClass span,
+        .ExternalClass font,
+        .ExternalClass td,
+        .ExternalClass div {
+          line-height: 100%; }
+        .apple-link a {
+          color: inherit !important;
+          font-family: inherit !important;
+          font-size: inherit !important;
+          font-weight: inherit !important;
+          line-height: inherit !important;
+          text-decoration: none !important; }
+        .btn-primary table td:hover {
+          background-color: #34495e !important; }
+        .btn-primary a:hover {
+          background-color: #34495e !important;
+          border-color: #34495e !important; } }
+
+    </style>
+  </head>
+  <body class="">
+    <table border="0" cellpadding="0" cellspacing="0" class="body">
+      <tr>
+        <td>&nbsp;</td>
+        <td class="container">
+          <div class="content">
+
+            <!-- START CENTERED WHITE CONTAINER -->
+            <span class="preheader">Registrazione repository al portale Developer Italia.</span>
+            <table class="main">
+
+              <!-- START MAIN CONTENT AREA -->
+              <tr>
+                <td class="wrapper">
+                  <table border="0" cellpadding="0" cellspacing="0">
+                    <tr>
+                      <td>
+                        <p>Gentile {{amministrazione}},</p>
+                        <p>Ci è pervenuta la richiesta di aggiundere la URL {{url}} all'elenco dei repository del portale Developer Italia da parte di {{referente}}.</p>
```

Comment:
```
Ci è pervenuta la richiesta di aggiungere la URL {{url}} al portale Developers Italia come strumento di code hosting ufficiale utilizzato dalla vostra amministrazione per pubblicare il software a riuso (secondo quanto disposto dalla Linee Guida di Acquisizione e Riuso del Software di AgID, con riferimento all'art 68 e 69 del Codice dell'Amministrazione Digitale).

La richiesta è stata effettuata dal seguente referente: {{referente}}.

Se la richiesta è corretta, potete confermare la registrazione utilizzando il pulsante qui in basso:
```

Reply:
Ok, l'ho corretto.

Reply:
https://github.com/italia/onboarding.developers.italia.it/pull/10/commits/084fb01b4d089c691c854a8c89d93e7569893bd9